src/frontend/src/pages/Notifications.tsx

```diff
@@ -125,6 +129,11 @@ export default function NotificationsPage() {
     ];
   }, [unreadTable, readTable]);
 
+  if (!globalSettings.isSet('NOTIFICATIONS_ENABLE')) {
```

Comment:
How are we communicating system errors without this?

Reply:
Yes but one has to explicitly know that it exhausts and that there might be errors as there are no indicators before you explicitly enter that pane in admin ui.
Not sure this is helpful to new users